### ISSUE TITLE
[FEATURE_REQUEST] Streaming /sysgen

### ISSUE THREAD

**Issue by michwad** [SillyTavern/SillyTavern#2292]
### Have you searched for similar requests?

Yes

### Is your feature request related to a problem? If so, please describe.

/sysgen doesnt work for Text Completion - InfermaticAI. (Tho it works if i use API type: Aphrodite and use Infermatic endpoint. 
I get this in CMD:
    logprobs: null
    }
  ],
  usage: { completion_tokens: 467, prompt_tokens: 5291, total_tokens: 5758 }
}
Endpoint error: TypeError: Cannot read properties of undefined (reading 'content')
    at F:\ai\SillyTavern-Launcher\SillyTavern\src\endpoints\backends\text-completions.js:328:99
    at Array.map (<anonymous>)
    at F:\ai\SillyTavern-Launcher\SillyTavern\src\endpoints\backends\text-completions.js:328:61
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    
   T

### Describe the solution you'd like

I would be awesome it it could be fixed. But even more so if streaming reply was implemented into the solution!

### Priority

Low (Nice-to-have)

### Are you willing to test this on staging/unstable branch if this is implemented?

None

**Comment by Cohee1207** [comment]
Non-streaming gens for Infermatic were fixed in 1.12.0

**Comment by Cohee1207** [comment]
Streaming is only supported for character replies and impersonation. You can inject a temporary instruction with /inject however.
```
/input What to generate? |
/inject id=sysgen position=chat depth=0 scan=true role=system ephemeral=true |
/trigger await=true |
/inject id=sysgen {{noop}} |
```
